samples/ExampleGallery/Shared/GlowTextCustomControl.cs

```diff
@@ -142,7 +142,10 @@ protected override Size MeasureOverride(Size availableSize)
             var layout = CreateTextLayout(device, availableSize);
             var bounds = layout.LayoutBounds;
 
-            return new Size(Math.Min(availableSize.Width, bounds.Width + ExpandAmount), Math.Min(availableSize.Height, bounds.Height + ExpandAmount));
+            Size desiredSize = new Size(Math.Min(availableSize.Width, bounds.Width + ExpandAmount), Math.Min(availableSize.Height, bounds.Height + ExpandAmount));
+            canvas?.Measure(desiredSize);
```

Comment:
Fixed. I am already so used to these new features that I don't even notice them anymore.